schemas/yaml/LexemeForm.yml

```
@@ -252,7 +257,8 @@ properties:
     description: If this form is a variant of another lexeme form, this field can be used to specify the type of variant. Possible values might be a person’s name (representing an idiolectal variant), or simply `idiolectal`, or `dialectal` (or the name of the dialect), or `rapid speech`, etc. Optionally in multiple languages.
 
 examples:
-  - allomorphs:
+  - abstract: False
```

```suggestion
  - abstract: false
```

Writing this as `False`  (initial caps) means that YAML will treat it as a string. Writing it as `false` (lowercase) means YAML and JavaScript will treat it like a Boolean. If you wrote `False` as a value in JavaScript, without quotes, the JS parser would throw an error, because it would be looking for a variable called `False`.